packages/pkg-config-openbsd/pkg-config-openbsd.1.0/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger"]
+homepage: "http://www.freedesktop.org/wiki/Software/pkg-config/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL"
+build: [
+  ["pkgconf" "--help"] {os = "openbsd"}
```

Comment:
I wonder if instead of having `os = "openbsd"` everywhere we could just add `available: [os = "openbsd"]` once?

Reply:
That would be one small improvement, if that works.

Reply:
I factorized the OS test thing.